Solaris/Illumos does not have sysfs support for pci bus
--- gecko-dev-master/toolkit/xre/glxtest/glxtest.cpp.~1~        2023-04-16 20:47:54.878612537 -0400
+++ gecko-dev-master/toolkit/xre/glxtest/glxtest.cpp    2023-04-16 20:50:01.983196670 -0400
@@ -264,11 +264,13 @@
 static void get_pci_status() {
   log("GLX_TEST: get_pci_status start\n");
 
+#if !defined(__sun)
   if (access("/sys/bus/pci/", F_OK) != 0 &&
       access("/sys/bus/pci_express/", F_OK) != 0) {
     record_warning("cannot access /sys/bus/pci");
     return;
   }
+#endif
 
   void* libpci = dlopen("libpci.so.3", RTLD_LAZY);
   if (!libpci) {
